configuration/pih/liquibase/sql/fix_migrate_non_coded_mch_dxs.sql

```diff
@@ -0,0 +1,165 @@
+    ('Phase expulsive du travail',concept_from_mapping('CIEL','163506')),
+    ('algie pelvienne',concept_from_mapping('CIEL','131034')),
+    ('IGU prob.',concept_from_mapping('PIH','15052')),
+    ('IGU pb',concept_from_mapping('PIH','15052')),
+    ('epigastralgie',concept_from_mapping('CIEL','141128')),
+    ('IGU',concept_from_mapping('PIH','15052')),
+    ('uc1',concept_from_mapping('CIEL','168593')),
+    ('IGU prob',concept_from_mapping('PIH','15052')),
+    ('UC2',concept_from_mapping('CIEL','168594')),
+    ('UTERUS MYOMATEUX',concept_from_mapping('CIEL','123455')),
+    ('Mycose vaginale',concept_from_mapping('CIEL','298')),
+    ('Uc1 ancien',concept_from_mapping('CIEL','168593')),
+    ('HTA chronique',concept_from_mapping('CIEL','168592')),
+    ('PES stab.',concept_from_mapping('CIEL','168611')),
+    ('uterus polymyomateux',concept_from_mapping('CIEL','123455')),
+    ('IGU probable',concept_from_mapping('PIH','15052')),
+    ('Phase expulsive',concept_from_mapping('CIEL','163506')),
+    ('Uc1 recent',concept_from_mapping('CIEL','168593')),
+    ('SUA',concept_from_mapping('CIEL','141631')),
+    ('Infection genito-urinaire',concept_from_mapping('PIH','15052')),
+    ('Infection génito urinaire pb',concept_from_mapping('PIH','15052')),
+    ('OMPK synd',concept_from_mapping('CIEL','129569')),
+    ('Algie pelvienne chronique',concept_from_mapping('CIEL','155579')),
+    ('Mycose vag',concept_from_mapping('CIEL','298')),
+    ('ASCUS',concept_from_mapping('CIEL','145822')),
+    ('suivi post op',concept_from_mapping('CIEL','159007')),
+    ('BDCFNR',concept_from_mapping('CIEL','168595')),
+    ('INFECTION GENITO URINAIRE',concept_from_mapping('PIH','15052')),
+    ('infertilite primaire',concept_from_mapping('CIEL','129123')),
+    ('OMPK',concept_from_mapping('CIEL','129569')),
+    ('Perimenopause',concept_from_mapping('CIEL','160595')),
+    ('IGU a inv',concept_from_mapping('PIH','15052')),
+    ('Masse ovarienne',concept_from_mapping('CIEL','152755')),
+    ('PES stabilisée',concept_from_mapping('CIEL','168611')),
+    ('Infertilite secondaire',concept_from_mapping('CIEL','126976')),
+    ('Infection génito urinaire',concept_from_mapping('PIH','15052')),
+    ('HTAc',concept_from_mapping('CIEL','168592')),
+    ('gardnerella vaginalis',concept_from_mapping('CIEL','139633')),
+    ('PES prob.',concept_from_mapping('CIEL','113006')),
+    ('AGUS',concept_from_mapping('CIEL','155209')),
+    ('PCOS',concept_from_mapping('CIEL','129569')),
+    ('HTA non controlee',concept_from_mapping('CIEL','165587')),
+    ('anamnios',concept_from_mapping('CIEL','168588')),
+    ('BDCF non rassurant',concept_from_mapping('CIEL','168595')),
+    ('uc2 ancien',concept_from_mapping('CIEL','168594')),
+    ('anemie probable',concept_from_mapping('CIEL','121629')),
+    ('douleur epigastrique',concept_from_mapping('CIEL','141128')),
+    ('PES stab',concept_from_mapping('CIEL','168611'));
+
+-- sanity check, should be 0, do not continue if not
```

Comment:
Seems like a typo here?  Do you mean _should not be 0_, or are you meaning to check a different table, or are you meaning to check that there are no rows with value_coded = null